contracts/KyberReserve.sol

```diff
@@ -4,21 +4,22 @@ pragma solidity 0.4.18;
 import "./ERC20Interface.sol";
 import "./Utils.sol";
 import "./Withdrawable.sol";
-import "./ConversionRates.sol";
+import "./ConversionRatesInterface.sol";
 import "./VolumeImbalanceRecorder.sol";
```

Comment:
while you're at this.
remove this one as well. (volume imbalance recorder...)

Reply:
Please submit pr for that.